test/integration/common/StudioActions.kt

```diff
@@ -210,11 +192,13 @@ object Utils {
     }
 
     suspend fun verifyDataWrite(composeRule: ComposeContentTestRule, address: String, dbName: String, queryFileName: String) {
-        val queryString = fileNameToString(queryFileName)
+        val queryString = Files.readAllLines(Paths.get(queryFileName), StandardCharsets.UTF_8)
```

Comment:
Why did we inline the function `fileNameToString`? I think it was great for the readability of `verifyDataWrite` to have the logic encapsulated. Instead of inlining the function, I would rather rename it to be clearer about its functionality. Something like "readQueryFileToString".

Also, why filter out the commented lines? I'd assume the functionality is the same with or without them.

Worth noting this will not filter out all _comments_, as a comment can be inserted in the middle of a line as well.

Reply:
Agh, I forgot to rejoin with `\n`.

Now that it isn't so specific, `readQueryFileToString` makes sense. I've reimplemented it as such.